app/src/main/java/com/example/javadevelopersnairobi/javadevelopersnairobi/model/Data.java

```diff
@@ -0,0 +1,46 @@
+package com.example.javadevelopersnairobi.javadevelopersnairobi.model;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.List;
+
+public class Data {
+//	@SerializedName("GithubUsersResponse")
+//	private GithubUsersResponse githubUsersResponse;
+//
+//	public GithubUsersResponse getGithubUsersResponse(){
+//		return githubUsersResponse;
+//	}
```

Comment:
Remove this commented code.

Reply:
Thank you for the heads ups, am going to remove them.